ietf/static_src/css/iab.scss

```diff
@@ -0,0 +1,200 @@
+    background-color: $blue-2;
+    border-radius: 4px;
+    padding: 16px;
+
+    &:hover {
+        background-color: $navy;
+    }
+}
+
+.home_page__jumbotron-iab {
+    // copied
+    background-position: 50%;
+    background-repeat: no-repeat;
+    background-size: cover;
+    position: relative;
+    &:before {
+        // second color is $brand-blue with 0% opacity
+        background-image: linear-gradient(
+            180deg,
+            $brand-blue 17.96%,
+            rgba(62, 150, 200, 0) 100%
+        );
+        // copied
+        content: '';
+        bottom: 0;
+        left: 0;
+        pointer-events: none;
+        position: absolute;
+        right: 0;
+        top: 0;
+    }
+}
+
+.bg-brand-blue-iab {
+    --bs-bg-opacity: 1;
+    background-color: $brand-blue !important;
+}
+
+.bg-blue-2-iab {
+    --bs-bg-opacity: 1;
+    background-color: $blue-2 !important;
+}
+
+.bg-navy-iab {
+    --bs-bg-opacity: 1;
+    background-color: $navy !important;
+}
+
+.btn-brand-blue-navbar-iab {
+    background-color: $brand-blue;
+    border-radius: 4px;
+
+    &:hover {
+        background-color: $blue-2;
+    }
+}
+
+.bg-navy-iab {
+    --bs-bg-opacity: 1;
+    background-color: $navy !important;
+}
+
+.logo-text-iab {
+    max-width: 124px;
+    font-size: 15px;
+    line-height: 17px;
+}
+
+.navbar-text-iab {
+    font-size: 14px;
+    line-height: 16px;
+}
+
+.footer-link-iab {
+    border-radius: 4px;
+
+    &:hover {
+        background-color: $blue-2;
+    }
+}
+
+.iab-blog-post {
+    a {
+        color: $brand-blue !important;
+        &:hover {
+            color: $blue-2 !important;
+        }
+    }
+}
+
+.card-text-iab {
+    font-size: 16px;
+    line-height: 21px;
+    /* or 131% */
+    letter-spacing: 0.4px;
+    font-weight: 300;
+}
+
+.date-text-iab {
+    color: $navy;
+    font-size: 14px;
+    line-height: 18px;
+    /* identical to box height, or 129% */
+    letter-spacing: 0.4px;
+    font-weight: 300;
+
+    p {
+        &:last-child {
+            margin-bottom: 0rem !important;
+        }
+    }
+}
+
+.card-iab {
+    padding: 1.5rem;
+}
+
+.blog-post-image-iab {
+    max-height: 12.5rem;
+}
+
+.iab-left-column {
+    margin-bottom: 4rem; //only on smallest size
+
+    @include media-breakpoint-up(lg) {
+        // background-color: skyblue;
+        margin-bottom: 0rem;
+        padding-right: 1.75rem;
+    }
+
+    @include media-breakpoint-up(xl) {
+        // background-color: orange;
+        margin-bottom: 0rem;
+        padding-right: 3rem;
+    }
+}
+
+.container-iab {
+    @include media-breakpoint-up(lg) {
+        max-width: 864px;
+    }
+
+    @include media-breakpoint-up(xl) {
+        max-width: 1000px;
+    }
+}
+
+.nav-item-iab {
+    border-radius: 4px;
+
+    &:hover {
+        .btn {
+            background-color: $blue-2;
+            // border-radius: 36px;
+        }
+    }
+
+    .dropdown-menu-iab {
+        margin-top: 36px !important;
+        border-radius: 4px;
+    }
```

Comment:
Does this selector need to be nested?

Reply:
as per our discussion, trying to keep it specific